bundles/io/org.eclipse.smarthome.io.rest.core/OSGI-INF/configurableserviceresource.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" name="org.eclipse.smarthome.io.rest.core.service.ConfigurableServiceResource">
+   <implementation class="org.eclipse.smarthome.io.rest.core.service.ConfigurableServiceResource"/>
+   <service>
+      <provide interface="org.eclipse.smarthome.io.rest.RESTResource"/>
+      <provide interface="org.eclipse.smarthome.io.rest.core.service.ConfigurableServiceResource"/>

```

Comment:
The implementation class does provide themselves as interface?
Do you want to use the ConfigurableService interface?
If you would like to use the introduced marker, you have to add the "implements" markup to the class.